src/main/resources/lang/EN.yml

```diff
@@ -51,10 +51,14 @@ del-deleted: "Warp '%0%' succesfully deleted."
 del-deletedOther: "Warp '%0%' succesfully deleted for %1%."
 
 list-header: "Warps from %0%:"
-list-public: "&7Public:&r %0%."
+list-public:
+  - "[gray]Public:[reset] %0%."
```

Comment:
Would be good to add bold formatting to indicate it can be clicked: `[bold]%0%[/bold]`